src/routes/properties/propertiesRoute.ts

```diff
@@ -180,9 +202,17 @@ export function propertyGetById() {
 
     const property = await dbModel.findById(id);
 
-    return property
-      ? buildApiResponse(200, toResponse(property, (key) => imageUrlFormatter(key, staticDomain)))
-      : buildNotFound();
+    if (!property) {
+      return buildNotFound();
+    }
+
+    if (!property.isConfirmed) {
```

Comment:
Let's add a check: if the user is the creator of the property - return it to the user. The creator should be able to see his non-cinfirmed properties